import-service/handlers/import-file-parser.ts

```diff
@@ -18,7 +19,14 @@ export const importFileParser = async (event : S3Event) => {
 
       return new Promise((resolve, reject) => {
         s3Stream.pipe(csv())
-          .on('data', data => { console.log(data) })
+          .on('data', async data => {
```

Comment:
Event handlers are synchronous. They don't wait for the result of an asynchronous function. So the lambda can be terminated before the messages are sent.

Streams are asyncrounous. You can use something like a streamTap transformer to synchronize the stream reading and sending the messages:

```js
import { Transform, pipeline as _pipeline } from "stream";
import { promisify } from "util";
const pipeline = promisify(_pipeline);

function streamTap(fn) {
  return new Transform({
    objectMode: true,
    transform: async (data, encoding, done) => {
      try {
        await fn({ data, encoding });
      } catch (err) {
        done(err);
        return;
      }
      done(null, data);
    },
  });
}

// usage
await pipline(
    s3Stream,
    csv(),
    streamTap(({data}) => console.log(data)),
    streamTap(async ({data}) => {
        await sqs.sendMessage(...).promise();
    })
);
```